tailwind.config.js

```diff
@@ -51,6 +51,9 @@ export default {
           1: '#1a1a1a',
           2: '#000000',
         },
+        background: {
```

Comment:
사소한거긴한데 대문자로 맞추는건 어떤가요~~
```suggestion
        Background: {
```